packages/webapp/pages/account/notifications.tsx

```diff
@@ -419,7 +448,7 @@ const AccountNotificationsPage = (): ReactElement => {
           </>
         )}
       </div>
-      {isPushSupported && (
+      {true && (
```

Comment:
Intentional?

Reply:
Nope, just had it for local dev, will fix

Reply:
Fixed